content/applications/finance/fiscal_localizations/belgium.rst

```diff
@@ -0,0 +1,164 @@
+  the registration of POS orders. If users are not clocked in, they cannot make POS orders.
+  Likewise, they must click :guilabel:`Work Out` to clock out at the end of the session.
+
+.. warning::
+   Odoo :ref:`On-premise <belgium/on-premise>` is *not* certified.
+
+The Fiscal Data Module
+----------------------
+
+The :abbr:`FDM (Fiscal Data Module)`, or `black box
+<https://www.systemedecaisseenregistreuse.be/systemes-certifies>`_ is a government-certified device
+that works together with the Point of Sale application and saves all your POS orders information.
+Concretely, a **Hash** (:dfn:`unique code`) is generated for each POS order and added to its
+receipt. This allows the government to verify that all revenue is declared.
+
+.. note::
+   - We recommend the use of the Cleancash SC-B.
+   - Ensure your black box is approved by the Belgian Government. You can check the compliance of
+     your black box by visiting  `www.boîtenoire.be <https://www.xn--botenoire-r5a.be/>`_.
+
+.. seealso::
+   More information concerning the Fiscal Data Module can be found on `the official website
+   <https://www.systemedecaisseenregistreuse.be/>`_.
+
+Configuration
+~~~~~~~~~~~~~
+
+Before setting up your database to work with a FDM, ensure you have the following hardware:
+
+- a registered :ref:`black box <belgium/blackbox>` (go to
+  `www.boîtenoire.be <https://www.xn--botenoire-r5a.be/>`_ to order yours and to the `FPS website
+  <https://finances.belgium.be/E-services/registratiemodule_gks/registratiemodule_gerigstreerd_kassasysteem>`_
+  to register it);
+- a serial null modem cable per FDM (`example
+  <https://www.startech.com/Cables/Serial-Parallel-PS-2/DB9-DB25/10-ft-Cross-Wired-Serial-Null-Modem-Cable-DB9-FM~SCNM9FM>`__);
+- a serial-to-USB adapter per FDM (`example
+  <https://trendnet.com/products/proddetail.asp?prod=265_TU-S9>`__); and
+- An :ref:`IoT Box <belgium/iotbox>` (one IoT box per black box).
+
+.. _belgium/blackbox:
+
+Black box
+*********
+
+As a pre-requisite, activate the **black box module**. To do so, go to the :guilabel:`Apps`
+application, remove the :guilabel:`Apps` filter, search for the ``Belgian Registered Cash
+Register`` module (technical name: ``pos_blackbox_be``) and click :guilabel:`Activate`.
+
+.. image:: belgium/be-modules.png
+   :align: center
+   :alt: black box modules for belgian fiscal certification
+
+Once the module is activated, update your company info with your VAT number. To set it up, go to
+:menuselection:`Settings --> Companies --> Update Info`, and fill in the :guilabel:`VAT` field.
+Then, enter a national registration number for every staff member who will be operating the POS
+system. To do so, go to the :guilabel:`Employees` app and open an employee form. After that, go to
+the :menuselection:`HR settings tab --> Attendance` and fill in the :guilabel:`INSZ or BIS number`
+field.
+
+.. image:: belgium/bis-number.png
+   :align: center
+   :alt: ISNZ or BIS number field on employee form
```

Comment:
Did you mean user form?

Reply:
no no, I meant employee, every person using the POS isn't necessary a user, some of them are just employees
But did you ask because every person from staff using a POS that is configured with a black box must be a user?

Reply:
Any user that can open the pos needs his ISNZ or BIS number

Reply:
Answer from rhe: any employee that is registered to use the POS need a BIS number, they don't have to have a "user account", even though being a user with POS admin rights is necessary to open the session. 
If one of the employees doesn't have a bis number attributed, --> the POS won't open!